app/src/main/java/no/usn/mob3000/ui/screens/user/ProfileEditScreen.kt

```diff
@@ -123,7 +127,7 @@ fun ProfileEditScreen(
                     /* TODO: Use country-code to decide on which emoji to display. */
                     commonCountries.forEach { (code, name) ->
                         DropdownMenuItem(
-                            text = { Text(name) },
+                            text = { Text(name, color = MaterialTheme.colorScheme.onBackground) },
```

Comment:
Indentation. And [see this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).